.luacheckrc

```diff
@@ -4,6 +4,7 @@
 cache = true
 
 ignore = {
+  "122", -- Setting a read-only field of a global variable.
```

Comment:
Why globally ignore this? If there's a specific case, ignore only the specific case.

Reply:
this is for `vim.opt_local`